salt/modules/postgres.py

```diff
@@ -316,31 +315,29 @@ def _connection_defaults(user=None, host=None, port=None, maintenance_db=None,
         port = __salt__['config.option']('postgres.port')
     if not maintenance_db:
         maintenance_db = __salt__['config.option']('postgres.maintenance_db')
-    if password is None:
-        password = __salt__['config.option']('postgres.pass')
 
-    return (user, host, port, maintenance_db, password)
+    return (user, host, port, maintenance_db)
 
 
 def _psql_cmd(*args, **kwargs):
     '''
     Return string with fully composed psql command.
 
-    Accept optional keyword arguments: user, host and port as well as any
-    number or positional arguments to be added to the end of command.
+    Accepts optional keyword arguments: user, host, port and maintenance_db,
+    as well as any number or positional arguments to be added to the end of
+    the command.
     '''
     (user, host, port, maintenance_db, password) = _connection_defaults(
```

Comment:
If `_connection_defaults` doesn't return a password anymore, then it needs to be removed here as well.

Reply:
@blueyed Were you able to come back to this PR?

Reply:
Apparently not.

Feel free to push a fixup here (maybe even through the GitHub UI), but since I am commenting already, I'll do it now.